### ISSUE TITLE
Syntax Error. 

### ISSUE THREAD

**Issue by AkashiX** [eslint/eslint#8497]
I get error when I write async/await keywrods with Vscode. Vscode integrates ESlint Plugin...Is it does not support for ES7 new syntax?

**Comment by eslintbot** [comment]
Hi @AkashiX, thanks for the issue. It looks like there's not enough information for us to know how to help you.

If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

Requesting a rule change? Please see [Proposing a Rule Change](http://eslint.org/docs/developer-guide/contributing/rule-changes) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by AkashiX** [comment]
Full Name: VS Code ESLint extension
Version: 1.2.8 ( find it at changelog in Vscode Editor )
There's many people say it wrong in Vscode Forum related.
Here's a sample:
```js
async create() {
  ...
  const promise = await this.ctx.service.userService.create(user);
  ...
}
```
I got error for "[eslint] Parsing error: Unexpected token create" on method name. But when I uninstall this pulgin, it works well. So I guess this plugin does not support for ES7 new syntax, such like above ( async/await ).

**Comment by not-an-aardvark** [comment]
Based on the information you've provided, it seems like this is an issue with the plugin and not an issue with ESLint.

**Comment by not-an-aardvark** [comment]
Closing because I think it's unlikely that this is an issue with ESLint.
